packages/server/api/routes/users.router.js

```diff
@@ -27,4 +27,47 @@ router.get('/', (req, res, next) => {
     .catch(next);
```

Comment:
I don't think this is needed so I would remove this route and the `getUsers` controller function.

Reply:
ok, it was just for testing purposes